packages/common/src/lib/entity/entity-table/entity-table.component.scss

```
@@ -65,4 +65,8 @@
   input {
     border-bottom: 1px solid darkgrey;
   }
+
+  tr.mat-mdc-header-row {
+    background-color: #f6f6f6;
+  }
```

Idéalement pour overider les couleurs, typo, density, on devrait le faire via un fichier de theming `entity-table.theming.scss`. Ce qui nous permettra d'honorer plus facilement les thèmes et un jour p-t pouvoir passé au DarkMode.
```suggestion
    $background: map-get($theme, background);

    tr.mat-mdc-header-row {
      background-color: mat.get-color-from-palette($background, background); // Il existe différente variante pour ajuster le contraste
    }
    ```